frontend/src/modules/studySamples/reducers.ts

```diff
@@ -131,10 +146,19 @@ export const studySamplesReducer = (state: WritableDraft<StudySamplesState>, act
 				for (const stepOrderID of stepOrderIDs) {
 					studyUXSettings.stepSettings[stepOrderID] = {
 						stepOrderID,
+						pageSize: 10
```

Comment:
again this hard-coded pagesize ... frontend/src/config.js has DEFAULT_PAGINATION_LIMIT that could serve that purpose ... (or you can add DEFAULT_SMALL_PAGINATION_LIMIT if you wish)